lib/commands/publish.js

```diff
@@ -157,12 +157,17 @@ class Publish extends BaseCommand {
       }
     }
 
-    const latestVersion = await this.#latestPublishedVersion(resolved, registry)
+    const newVersion = manifest.version
+    const { latest: latestVersion, versions } = await this.#registryVersions(resolved, registry)
```

Comment:
> const latestSemverIsGreater = !!latestVersion && semver.gte(latestVersion, manifest.version)

I can't leave a comment on this line, but you may want to switch to `semver.gt` for consistency with the error message.

Reply:
adjusting